File: java/TuduManager/repository.postgres.sql2o/src/main/java/net/seymourpoler/tudumanager/repository/postgres/sql2o/user/save/SaveUserRepository.java
package net.seymourpoler.tudumanager.repository.postgres.sql2o.user.save;

import net.seymourpoler.tudumanager.domain.user.signUp.ISaveUserRepository;
import net.seymourpoler.tudumanager.domain.user.signUp.Models.User;
import net.seymourpoler.tudumanager.repository.postgres.sql2o.DataBaseConnectionFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

@Component
public class SaveUserRepository implements ISaveUserRepository {

    private final DataBaseConnectionFactory dataBaseConnectionFactory;

    @Autowired
    public SaveUserRepository(DataBaseConnectionFactory dataBaseConnectionFactory) {
        this.dataBaseConnectionFactory = dataBaseConnectionFactory;
    }

    @Override
    public void save(User user) {
        final String sql = "INSERT INTO public.users (email, passord) VALUES (:email, :password, :creation_date)";
        var dbModel = buildFrom(user);
        try (var connection = dataBaseConnectionFactory.create()) {
            connection
                .createQuery(sql)
                .bind(dbModel)
                .executeUpdate();
        }
    }

    private net.seymourpoler.tudumanager.repository.postgres.sql2o.user.save.models.User buildFrom(User user) {
        var result = new net.seymourpoler.tudumanager.repository.postgres.sql2o.user.save.models.User();
        result.email = user.email;
        result.password = user.password;
        result.creation_date = user.creationDate;
        return result;
    }
}

File: dotnet/react.mvc-core.postgres.dapper/TuduManager/src/TuduManayer.Repository.Postgres.EntityFramework/User/SaveUserRepository.cs
using TuduManayer.Domain.User.SignUp;

namespace TuduManayer.Repository.Postgres.EntityFramework.User
{
    public class SaveUserRepository : ISaveUserRepository
    {
        private readonly DataBaseContextFactory dataBaseContextFactory;

        public SaveUserRepository(DataBaseContextFactory dataBaseContextFactory)
        {
            this.dataBaseContextFactory = dataBaseContextFactory;
        }

        public void Save(Domain.User.User user)
        {
            using var dbContext = dataBaseContextFactory.Create();
            dbContext.Users.Add(new Models.User
            {
                email = user.Email,
                password = user.Password,
                creation_date = user.CreationDate,
                updated_date = null
            });
            dbContext.SaveChanges();
        }
    }
}